### PR TITLE
[v6r20] fixed upload of pilot JSON file for pilot 3

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Configuration
FIX: fixes for enabling pilotCStoJSONSynchronizer, and doc

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Review, please

File: ConfigurationSystem/Service/ConfigurationHandler.py [hunk 1]
```diff
@@ -3,109 +3,109 @@
...
-    if not 'DN' in credDict or not 'username' in credDict:
-      return S_ERROR( "You must be authenticated!" )
-    res = gServiceInterface.updateConfiguration( sData, credDict[ 'username' ] )
-
-    if self.updatePilotJSONFile:
-      if not res['OK']:
-        return res
-      return PilotCStoJSONSynchronizer( self.paramDict ).sync()
-    else:
+    if 'DN' in credDict or not 'username' not in credDict:
```
Comment (zmathe) [inline ConfigurationSystem/Service/ConfigurationHandler.py]: not 'username' not in credDict?

Comment (fstagni) [reply ConfigurationSystem/Service/ConfigurationHandler.py]: yes, corrected, thanks.

File: ConfigurationSystem/Service/ConfigurationHandler.py [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
-    if not 'DN' in credDict or not 'username' in credDict:
-      return S_ERROR( "You must be authenticated!" )
-    res = gServiceInterface.updateConfiguration( sData, credDict[ 'username' ] )
-
-    if self.updatePilotJSONFile:
-      if not res['OK']:
-        return res
-      return PilotCStoJSONSynchronizer( self.paramDict ).sync()
-    else:
+    if 'DN' in credDict or 'username' not in credDict:
```
Comment (andresailer) [inline ConfigurationSystem/Service/ConfigurationHandler.py]: Comparing with the old version: It should also be `'DN' not in credDict`

Comment (fstagni) [reply ConfigurationSystem/Service/ConfigurationHandler.py]: indeed! fixed that too

**Comment by atsareg** [comment]
Merging, will fix the reported bug while tagging for the release

**Pull request merged**